docs/applications.md

```diff
@@ -8,35 +8,51 @@ In order to deploy application you can use `eden pod deploy` command:
---vnc-password string   VNC password (empty - no password)
---volume-type string    volume type for empty volumes (qcow2, raw, qcow, vmdk, vhdx or oci); set it to none to not use volumes (default "qcow2")
+      --acl strings           Allow access only to defined hosts/ips/subnets
+                              You can set acl for particular network in format '<network_name:acl>'
+                              To remove acls you can set empty line '<network_name>:'
+      --adapters strings      adapters to assign to the application instance
+      --cpus uint32           cpu number for app (default 1)
+      --direct                Use direct download for image instead of eserver (default true)
+      --disk-size string      disk size (empty or 0 - same as in image) (default "0 B")
+      --disks strings         Additional disks to use. You can write it in notation <link> or <mount point>:<link>. Deprecated. Please use volumes instead.
+      --format string         format for image, one of 'container','qcow2','raw','qcow','vmdk','vhdx'; if not provided, defaults to container image for docker and oci transports, qcow2 for file and http/s transports
+  -h, --help                  help for deploy
+      --memory string         memory for app (default "1.0 GB")
+      --metadata string       metadata for pod
+  -n, --name string           name for pod
+      --networks strings      Networks to connect to app (ports will be mapped to first network)
+      --no-hyper              Run pod without hypervisor
+      --only-host             Allow access only to host and external networks
+      --openstack-metadata    Use OpenStack metadata for VM
+  -p, --publish strings       Ports to publish in format EXTERNAL_PORT:INTERNAL_PORT
+      --registry string       Select registry to use for containers (remote/local) (default "remote")
+      --sftp                  Force use of sftp to load http/file image from eserver
+      --vnc-display uint32    display number for VNC pod (0 - no VNC)
+      --vnc-password string   VNC password (empty - no password)
+      --volume strings        Additional volumes to use. You can write it in notation <link> or <mount point>:<link>.
+      --volume-size string    volume size (default "200 MiB")
+      --volume-type string    volume type for empty volumes (qcow2, raw, qcow, vmdk, vhdx or oci); set it to none to not use volumes (default "qcow2")
 
 Global Flags:
---config string      Name of config (default "default")
--v, --verbosity string   Log level (debug, info, warn, error, fatal, panic (default "info")
+      --config string      Name of config (default "default")
+  -v, --verbosity string   Log level (debug, info, warn, error, fatal, panic (default "info")
 ```
 
+### Mount
+
+You can pass additional volumes to application with `--volume` flag. For example:
+
+```bash
+eden pod deploy docker://itmoeve/eclient:0.7 -p 8027:22 --volume=/tst:docker://nginx:1.20.0 --volume=/dir:directory://./tests
```

Comment:
may we support the docker syntax
 -v nginx-vol:/usr/share/nginx/html
for us will be: 
--volume docker://nginx-vol:/usr/share/nginx/html
or 
--volume /my/nginx:/usr/share/nginx/html

So if we don't have a prefix we assume it's a local dir

Reply:
we may also wanna support some existing volumes?

Reply:
Sorry, can you please elaborate about proposed existing volumes?

Reply:
I mean we have the idea of volumes in Zcloud. You create it and then can list and mount.